### PR TITLE
Initial version of Healthcare API client

### PR DESCRIPTION
**Pull request by davidfarkas**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.vscode
```
Comment (ambrussimon) [inline .gitignore]: Please sort.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM python:3-alpine
+
+RUN apk update \
+    && apk add build-base jpeg-dev zlib-dev \
```
Comment (ambrussimon) [inline Dockerfile]: Are all those libs still used somewhere?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+version_filename = os.path.join(package_root, 'healthcare_api', 'VERSION')
+with open(version_filename) as f:
+    version = f.read().strip()
+
+setup(
+    name=NAME,
+    version=version,
+    description="Google Cloud Healthcare API client library",
+    author_email="support@flywheel.io",
+    url="",
```
Comment (ambrussimon) [inline setup.py]: Isn't this where the VCS url usually goes?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    keywords=["Flywheel", "flywheel", "google", "healthcare", "api"],
+    install_requires=INSTALL_REQUIRES,
+    extras_require={
```
Comment (ambrussimon) [inline setup.py]: Maybe `dicomweb-client` could be listed as an extras_require dependency?

File: Dockerfile (hunk 2 above)
Comment (davidfarkas) [reply Dockerfile]: I will remove the whole docker file, since it is not used anymore.

**Pull request merged**